src/core/context.ts

```diff
@@ -63,8 +63,10 @@ export function createRoutesContext(options: Required<Options>) {
 
   function addPage(path: string) {
     log('added', path)
+    const route = stripRouteFolder(path)
+    // TODO: handle top level named view HMR
```

Comment:
what did you mean with this?

Reply:
I haven't dig into it yet, but currently named view has some bug with HMR when you change routes's filename.